Libraries/Geolocation/Geolocation.ios.js

```diff
@@ -23,11 +23,9 @@ var subscriptions = [];
 var updatesEnabled = false;
 
 /**
- * /!\ ATTENTION /!\
- * You need to add NSLocationWhenInUseUsageDescription key
- * in Info.plist to enable geolocation, otherwise it's going
- * to *fail silently*!
- * \!/           \!/
+ * You need to include the `NSLocationWhenInUseUsageDescription` key
+ * in Info.plist to enable geolocation. Geolocation is enabled by default
+ * when you create a project with `react-native init`.
```

Comment:
@spicyj - interesting point, you guys probably have more expertise here so I'll defer to you - I can revert this if you prefer.